prep-css-display/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Display Properties</title>
+  <style>
+    button {
+      border: 4px solid red;
+      height: 40px;
+      width: 200px;
+    }
+
+    li {
+      border: 4px solid magenta;
+      display: inline;
+    }
+  </style>
+</head>
+
+<body>
+  <div style="border: 4px solid blue;"'>
+    I am a div and by default I am block, which means I take up all the space I can horizontally.
+  </div>
+  <div style="border: 4px solid blue;">
+    I am a div but I have <span style="border: 4px solid green;">span elements inside me</span>.
+    Span elements are <span style="border:4px solid green">display inline by default</span>,
+    so they only take up as much as space as they need.
+  </div>
+  <div style="border: 4px solid blue;">
+    <button>Hello!</button>
+    <button>Buttons are</button>
+    <button>Inline Block</button>
+    <button>By Default</button>
+  </div>
+  <ul style="border: 4px solid purple;">
+    <li>
```

Comment:
for this exercise, try doing all of your styling in the CSS code in your `<style></style>` element.